go/worker/storage/committee/node.go

```diff
@@ -295,15 +297,25 @@ func NewNode(
 					ChunkSize: rt.Storage.CheckpointChunkSize,
 				}, nil
 			},
-			GetRoots: func(ctx context.Context, version uint64) ([]hash.Hash, error) {
+			GetRoots: func(ctx context.Context, version uint64) ([]storageApi.Root, error) {
 				blk, berr := commonNode.Runtime.History().GetBlock(ctx, version)
 				if berr != nil {
 					return nil, berr
 				}
 
-				return []hash.Hash{
-					blk.Header.IORoot,
-					blk.Header.StateRoot,
+				return []storageApi.Root{
+					{
+						Namespace: commonNode.Runtime.ID(),
```

Comment:
Why not `blk.Header.Namespace` to be consistent with below? And then you could add a common method to get the storage roots from the header -- actually there already is `blk.Header.StorageRoots()`, why don't you just use that?